modules/imaging_browser/README.md

```diff
@@ -0,0 +1,81 @@
+# Imaging Browser
+
+## Purpose
+
+The imaging browser is intended to allow users to view candidate
+scan sessions collected for a study.
+
+## Intended Users
+
+The three primary types of users are:
+1. Imaging specialists using the modules to do online QC
+2. Project radiologists viewing images to report incidental findings
+3. Site coordinators or researchers ensuring their uploaded scans have
+   been processed and inserted into LORIS.
+
+## Scope
+
+The imaging browser displays images processed images which meet the
+study's inclusion criteria. The inclusion criteria (for most images)
+is defined and enforced in the LORIS imaging pipeline scripts.  Derived
+or processed scan types are also included and have their own insertion
+criteria.
+
+NOT in scope:
+
+The imaging browser module does not display raw DICOMs or perform automated
+quality control on images. It only displays images that have already been
```

Comment:
Your statement is correct here. But just a small comment; the SNR automatically computed by Loris-MRI and displayed in Imaging Browser is considered a form of automated QC. Sao while imaging browser does not perform this automated QC, it displays some... is it worth mentioning or no?

Reply:
I don't know if it's worth mentioning. It seems like an implementation detail of what's displayed and is displayed for the purpose of helping manual qc. It doesn't affect the scope of the module, since the calculation of the SNR is done outside of the module by the pipeline scripts.